docs/content/快速上手/tidb-tutorial-zh.md

```diff
@@ -232,7 +232,7 @@ Flink SQL> INSERT INTO enriched_orders
 INSERT INTO orders
 VALUES (default, '2020-07-30 15:22:00', 'Jark', 29.71, 104, false);
 
-UPDATE orders SET order_status = true WHERE order_id = 10004;
+UPDATE orders SET customer_name = 'Aiden' WHERE order_id = 10004;
```

Comment:
Do we have to change the update column? I think make it unified with other connector tutorial is more friendly to users.